pkg/auth/cleanup/service.go

```diff
@@ -181,6 +191,29 @@ func (s *Service) deleteUsers(config *v3.AuthConfig) error {
 	return nil
 }
 
+// deleteTokens deletes all the tokens created with the disabled provider
+func (s *Service) deleteTokens(config *v3.AuthConfig) error {
+	if config == nil {
+		return errAuthConfigNil
+	}
+
+	tokens, err := s.tokensCache.List(labels.Everything())
+	if err != nil {
+		return fmt.Errorf("failed to list tokens: %w", err)
+	}
+
+	for _, t := range tokens {
+		if t.AuthProvider == config.Name {
+			err := s.tokensClient.Delete(t.Name, &metav1.DeleteOptions{})
+			if err != nil && !apierrors.IsNotFound(err) {
+				return err
```

Comment:
How about we wrap this with `"cleaning up authprovider %s: %w", t.AuthProvider, err)` or something similar to help debug issues?

Reply:
Sure, wrapping is always important.